### PR TITLE
doc: nrf: Update documentation for multi-image builds in SES

### PR DESCRIPTION
**Pull request by sigvartmh**
The workflow of building a multi-image in SES is a bit different form
just building a standard sample. This commit adds the build steps
required for building a multi-image sample/application with SES.

Signed-off-by: Sigvart Hovland <sigvart.hovland@nordicsemi.no>

### CODE REVIEW

**Comment by sigvartmh** [comment]
@ru-fu if you have time could you look at it? I'm not completely sure how you want to add this. Also maybe it should give an example of a sample which uses the multi-image solution.

**Comment by sigvartmh** [comment]
@osaether documentation update for using SES

File: doc/nrf/gs_programming.rst [hunk 1]
```diff
@@ -58,18 +58,31 @@ You can download it from the following links:
...
 	   the build completes.
 
-6. Once the build completes, you can flash the sample to a connected board.
-To do this, select **Target -> Download zephyr/zephyr.elf**.
+   To build a multi-image project:
+
+       a. Select **Build -> Build Solution**.
+       #. When the multi-image build completes, you can flash the sample to a connected board.
+          To do this, select **Target -> Download File -> Download Intel Hex File**.
+          Navigate to the ``zephyr`` folder in the build directory and select ``merged.hex``.
```
Comment (ru-fu) [inline doc/nrf/gs_programming.rst]: Is that what you meant? If not, can you send me a screenshot of that window?

**Comment by ru-fu** [comment]
> 
> 
> Also maybe it should give an example of a sample which uses the multi-image solution.

Yes, for sure. Which samples are using it?

File: doc/nrf/gs_programming.rst (hunk 1 above)
Comment (sigvartmh) [reply doc/nrf/gs_programming.rst]: Will do :)

**Comment by sigvartmh** [comment]
> > Also maybe it should give an example of a sample which uses the multi-image solution.
> 
> Yes, for sure. Which samples are using it?

The only ones I can think of now is something with SPM enabled or MCUBoot

**Comment by sigvartmh** [comment]
> > > Also maybe it should give an example of a sample which uses the multi-image solution.
> > 
> > 
> > Yes, for sure. Which samples are using it?
> 
> The only ones I can think of now is something with SPM enabled or MCUBoot

So anything building with nrf9160_pca10056ns will use multi-image I think. At least hello-world does.

File: doc/nrf/gs_programming.rst (hunk 1 above)
Comment (sigvartmh) [reply doc/nrf/gs_programming.rst]: Done

**Comment by ru-fu** [comment]
Do we need to update the build instructions for the nRF9160 samples? See for example here: https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/samples/nrf9160/coap_client/README.html#building-and-running

**Comment by sigvartmh** [comment]
> Do we need to update the build instructions for the nRF9160 samples? See for example here: https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/samples/nrf9160/coap_client/README.html#building-and-running

This old workflow should still works. However you could also build the sample by simply selecting `nrf9160_pca10056ns` in the sample and it would automatically include `SPM + Application` into `merged.hex` however I'm a bit unsure about what configurations are set for SPM but I guess it should be the same as it was before.

**Comment by sigvartmh** [comment]
> Do we need to update the build instructions for the nRF9160 samples? See for example here: https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/samples/nrf9160/coap_client/README.html#building-and-running

@ru-fu Actually I found that you could just select the `zephyr/merged.hex` project select it with set as active project. Then you can flash it with the same workflow as described before.

**Comment by ru-fu** [comment]
I think I need to see that to understand what you mean. ;-)
If the current description works, maybe we should merge this as is and update the sample descriptions in a separate PR later?

**Comment by sigvartmh** [comment]
Yeah, that's fine 👍 We could do another revision on the sample descriptions.

**Pull request merged**